kuma/javascript/src/payments/list-item.jsx

```diff
@@ -0,0 +1,20 @@
+//@flow
+import * as React from 'react';
+
+import { gettext } from '../l10n.js';
+
+type ListItemProps = {
+    title: string,
+    number: string,
+    children: React.Node
+};
+
+const ListItem = ({ title, number, children }: ListItemProps) => (
+    <li className="faq">
+        <h3>{gettext(title)}</h3>
```

Comment:
Re: localization-- this is the part I'm unsure about and trying to test. I thought this was simpler since it saves a few `gettext` calls, but I'm not sure if `gettext` will work with passing in text as a prop.